puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/foreman-x-develop-release.groovy

```diff
@@ -29,22 +29,6 @@ pipeline {
                 )
             }
         }
-        stage('Archive Git Commit') {
```

Comment:
This file is included in various jobs:
```
$ rg foreman-x-develop -l
puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/release/foreman-develop-release.yaml
puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/release/smart-proxy-develop-release.yaml
puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/release/foreman-installer-develop-release.yaml
puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/release/hammer-cli-foreman-master-release.yaml
puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/release/hammer-cli-master-release.yaml
puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/release/katello-master-release.yaml
puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/release/foreman-selinux-develop-release.yaml
puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/release/hammer-cli-katello-master-release.yaml
```

Which is why I suggested these all need to be updated.

Reply:
Duh. That makes complete sense. Since there is no harm in this existing till all are converted I put these lines back in this fileto migrate paradigms one by one.